screen/lnd/LNDViewAdditionalInvoiceInformation.tsx

```
@@ -0,0 +1,108 @@
+    root: {
+      backgroundColor: colors.elevated,
+    },
+  });
+
+  useEffect(() => {
+    if (wallet) {
+      wallet
+        .fetchInfo()
+        .then(() => {
+          const info = wallet.info_raw;
+          if (info && info.uris && info.uris[0]) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)